### DISCUSSION TITLE
(Taiko?) Mismatched score info in results / replay / profile

### DISCUSSION THREAD

**Discussion by 12beesinatrenchcoat** [ppy/osu#19917] [Q&A] [answered]
Played some osu!taiko, set a score, and checked my profile — the accuracy shown on my profile and in-game don't match up. 
![image](https://user-images.githubusercontent.com/25379179/186021843-c0b8d4e1-7937-4b1b-b9ef-608612fe6a90.png)

Additionally, when exporting the replay and importing it, the accuracy of the replay matches the online accuracy, but the score and strong bonus change.

https://user-images.githubusercontent.com/25379179/186022880-d7c2bca0-6126-4378-ac02-e4e6dd9e5761.mov

oh yeah logs 
[runtime.log](https://github.com/ppy/osu/files/9397742/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9397743/updater.log)
[database.log](https://github.com/ppy/osu/files/9397744/database.log)
[network.log](https://github.com/ppy/osu/files/9397745/network.log)
[performance.log](https://github.com/ppy/osu/files/9397746/performance.log)

**Comment by smoogipoo** [comment]
This is likely going to be an osu!web issue, however it's likely that this will be addressed by the scoring changes I've got on the way combined with the ES indexer indexing accuracy? @nanaya / @notbakaneko probably just keep this one in the back of your mind.

**Comment by nanaya** [comment, marked as answer]
The scores displayed on osu-web are the legacy scores and lazer scores which are converted to legacy structure. It means the accuracy is calculated using older algorithm. There's no way to fix it (in its current form) until osu-web fully switches to new scores system.
